test/jest.setup.js

```diff
@@ -0,0 +1 @@
+jest.setTimeout(100000);
```

Comment:
This is bad ™️. Just for messing around, with fix with a better await timeout pattern